tutorials/text/pgf.py

```diff
@@ -187,6 +187,11 @@
   using either the ``rasterized=True`` keyword, or ``.set_rasterized(True)`` as
   per :doc:`this example </gallery/misc/rasterization_demo>`.
 
+* Various math font are compiled and rendered only if corresponding font
+  packages are loaded. The ``lmodern`` package is often required when using
+  math symbols. See `discussion <https://matrix.to/#/!BXmyZMTnRjWJldDRLV:gitter.im/$M7KxLQuyUosx2byFxIsKJE8QS_14NRiWqmmlb6nSIfE?via=gitter.im&via=matrix.org>`
```

Comment:
I don't think linking a gitter discussion (via matrix!) is something we should start doing in our docs.

Reply:
I agree, could you summarize the discussion here?  

We should try to, with in reason, keep our docs self-contained and usable by people in off-line situations (for example on an airplane or boat).

Reply:
I've made the discussion more specific.